src/__tests__/util.ts

```diff
@@ -62,6 +62,33 @@ export function createEmptyState(): AtemState {
 	}
 	state.fairlight = {
 		inputs: {},
+		master: {
+			equalizerBands: [undefined, undefined, undefined, undefined, undefined],
+			equalizerGain: 0,
+			equalizerEnabled: false,
+			makeUpGain: 0,
+			faderGain: 0,
+			followFadeToBlack: false,
+		},
+	}
+
+	for (let i = 0; i < 6000; i++) {
```

Comment:
6000 empty object seems like a lot :)

Reply:
its not pretty, but was necessary to be at lease 2000 for the tests to be happy.
Otherwise the tests running applyToState would fail at https://github.com/nrkno/tv-automation-atem-connection/blob/feat/bitfocus-commands/src/commands/Fairlight/FairlightMixerSourceCompressorCommand.ts#L82.

It will probably be possible to decrease that number (I dont remember how high the ids can go), but for the cost of at most a couple mb of ram during the tests I dont see the benefit

Reply:
I see this file is only used in testing, not as important as I thought then!

Reply:
After fixing a bug, I discovered that I also need to create the correct sources inside these, with the index being some hard to guess string. So I changed this to do some light inspection of the command it is for instead